### DISCUSSION TITLE
Trouble with Firefox Border Shine

### DISCUSSION THREAD

**Discussion by OldManAlpha** [Maplespe/DWMBlurGlass#290] [Q&A]
This is probably not related to DWMBlurGlass, but I figure people here may have encountered this issue. I have Firefox/Waterfox setup to where the entire window is glass using CustomCSSforFx(wish I knew a way to only get a smaller portion to be glass to improve performance). When using vaporvance's or alcatel's Windows 7 theme with the border shine effect on, it doesn't have any gradient and covers the entire upper portion of the window, making the whole title bar have an ugly white tint. Does anybody have any ideas on how to solve this issue?

Theme without shine:
![image](https://github.com/Maplespe/DWMBlurGlass/assets/60587722/6a456bf1-85fd-4cf3-85f3-2638aecccb7f)
Theme with shine:
![image](https://github.com/Maplespe/DWMBlurGlass/assets/60587722/fbe4b6ab-3ee4-491c-afbb-26c43ba00f4a)

**Comment by 7kt4** [comment]
There is a bug that if the glass sheet is very tall, then it stretches the borders from the theme across the window
No idea why it happens or how to fix
Only way is to not have it is to drop the ends shine :/

**Reply by OldManAlpha** [reply]
Is there any way to reduce the amount of glass Firefox places, or a way to exclude a specific program from having that shine effect. It's not the end of the world if Firefox doesn't have the shine, but it's a real shame to have to disable it for everything just because one program doesn't play well with it. Frankly looking for any crazy solutions there might be to just get rid of the fog on Firefox but keep the shine for everything else.
